kubeflow/mpi-job/prototypes/mpi-operator.jsonnet

```diff
@@ -0,0 +1,26 @@
+// @apiVersion 0.1
+// @name io.ksonnet.pkg.mpi-operator
+// @description MPI Operator.
+// @shortDescription MPI Operator for running MPI jobs.
+// @param name string Name to give to each of the components.
+// @optionalParam namespace string null Namespace to use for the components. It is automatically inherited from the environment if not set.
+// @optionalParam image string mpioperator/mpi-operator:latest The image for the MPI Operator.
+// @optionalParam kubectlDeliveryImage string mpioperator/kubectl-delivery:latest The image for delivering kubectl.
+// @optionalParam gpusPerNode number 8 The maximum number of GPUs per node.
+
```

Comment:
Sometimes, cluster-admin wanted to manage RBAC related resources on their own or to use existing service account.  In the case, admins don't want to let the prototype create them.  So, 

How about introducing `rbacCreate` and `serviceAccountName` parameters??  It's popular way in helm packages.  In this case, we need explicit guidance so that admins can refer to required roles when they choose `rbacCreate=false`.

Reply:
Good idea, but it'll involve some code changes. Future enhancement? Tracking it at https://github.com/kubeflow/mpi-operator/issues/20.